_posts/2023-07-31-paris-dev-sprint.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "scikit-learn 2023 In-person Developer Sprint in Paris, France"
+date: July 31, 2023
+
+categories:
+  - Events
+tags:
+  - Sprints
+  - Community
+
+postauthors:
+  - name: Reshama Shaikh
+    website: https://reshamas.github.io
+    image: reshama_shaikh.jpeg 
+  - name: François Goupil
+    email: francois.goupil@inria.fr
+    website: https://github.com/francoisgoupil
+    image: "francois_goupil.jpeg"
+---
+<div>
+  {% include postauthor.html %}
+</div>
+
+
+In June of 2023 (week of June 19 to 23), the scikit-learn team held its first developers sprint since 2019! The sprint took place in Paris, France at the Dataiku office. The sprint event was an in-person event only and had 29 participants.
+
+The following [scikit-learn team members](https://scikit-learn.org/stable/about.html) joined the sprint:  
+
+1. Gael Varoquaux (@GaelVaroquaux)
+1. Olivier Grisel (@ogrisel)
+1. Guillaume Lemaitre (@glemaitre)
+1. Tim Head (@betatim)
```

Comment:
```suggestion
1. Tim Head (@betatim)
1. Jérémie du Boisberranger (@jeremiedbb)
1. Frank Charras (@fcharras)
```

Reply:
@jjerphan Is Frank Charras a scikit-learn team member?

Reply:
Franck is working on GPU implementations with Olivier and Tim.

He should be part of scikit-learn team if he's not IMO.